Loop/View Controllers/BatteryTypeSelectionTableViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  BatteryTypeSelectionTableViewController.swift
+//  Loop
```

Comment:
I am sure it may be. I am just not swift enough to fully comprehend how.  So I leveraged the existing pattern Nate had provided.